src/main/resources/springintegration/flows.xml

```diff
@@ -9,6 +9,7 @@
     <int:annotation-config/>
 
     <import resource="case-gateway-event-inbound.xml"/>
+    <import resource="case-service-event-outbound.xml"/>
     <import resource="respondent-service-event-outbound.xml"/>
```

Comment:
remove - but look for vestigial refs

Reply:
Removed config xml file. 
Also removed some unused code which was using the rabbit template bean.